app/assets/javascripts/submit_form_ajax.js

```diff
@@ -10,11 +10,12 @@ function sendFormSubmissionAjax(dataObj, submitTo, responseEl = "") {
   }
   $.ajax({
     url: url,
-    data: dataObj,
-    success: (event, success) => {
-      if (responseEl !== "") {
-        $(responseEl).trigger('ajax:success', event);
-      }
+    data: dataObj
+  })
+  .done((event) =>{
+    if (responseEl !== "") {
```

Comment:
```suggestion
    if (!!responseEl) {
```

Reply:
ahh okay 👍 
Only thing is if `responseEl ` is undefined or null, then also this condition would be true, so it is better to use:
`if(!!x)` or `if(x)` for safer side

Reply:
I see, that makes sense... I'll ignore the CodeClimate warning.